inginious/frontend/plugins/multilang/problems/custom_input.py

```diff
@@ -44,11 +44,10 @@ def API_POST(self):
                     if type(value) is str:
                         user_input[key] = user_input[key].replace("\r", "")
                 result, grade, problems, tests, custom, archive, stdout, stderr = self.add_unsaved_job(task, user_input)
-
                 data = {
                     "status": ("done" if result[0] == "success" or result[0] == "failed" else "error"),
                     "result": result[0],
-                    "text": ParsableText(result[1]).parse(),
+                    "text": ParsableText(result[1],"rst").parse(),
```

Comment:
It is not clear to me why set the type of response to rst. Could you please clarify it?

Reply:
Yes, the answer is on INGInious containers repo, now the submission's feedback on multilang container is passed to the webapp on json by default if the task have not response_type field defined. But the thing is that the custom input feedback is managed different that the submission's feedback on the multilang container, so there are two options:
- Set the parser on the webapp to rst on multilang's custom input requests, this in order to not modify multilang's container code
- Set the parser based on the response type field, this need also changes on how multilang's container manage the custom input feedback but we consider this innecesary at the moment because this kind of requests are not saved on the database.

What approach do you think that is better?

Reply:
I agree that the fisrt approach is better as it is less intrusive. Thanks for the clarification. Good job!